WebAPI/CurrencyConverter/Data/ExchangeRateRepository.cs

```diff
@@ -0,0 +1,54 @@
+﻿using CurrencyConverter.Interfaces;
+using Model;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+
+namespace CurrencyConverter.Data
+{
+    public class ExchangeRateRepository : IExchangeRateRepository
+    {
+        protected readonly ExchangeRateContext _context;
```

Comment:
Why we make this field protected? Will there be classes which inherit from this repo?